endpoints/cps.yaml

```diff
@@ -223,6 +223,139 @@ paths:
             application/json:
               schema:
                 $ref: "../components/Error.yaml"
+  /v2/initialize:
+    post:
+      description: Initialize
+      operationId: initialize
+      parameters:
+        - in: header
+          name: Fantasmo-Key
+          schema:
+            type: string
+          required: true
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                coordinate:
```

Comment:
Is there any way to mark a field as required? I mean I know, technically, only one of either coordinate or location is required but yer, would make writing tests a lot easier if I could look at the spec file and be able to differentiate optional from required fields :)

Reply:
I believe there is: https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

Reply:
Added this feature for `/v1/image.localize` and `/v2/initialize`, please take a look

Reply:
@GrayedFox